src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -51,6 +53,16 @@ private static (RecalcEngine engine, RecordValue parameters) OptionSetTestSetup(
             return (new RecalcEngine(config), parameters);
         }
 
+        private static (RecalcEngine engine, RecordValue parameters) AsyncSetTestSetup()
```

Comment:
>AsyncSetTestSetup [](http://example.com/codeflow?start=69&length=17)

Each "setup" is removing the configuration from other setups... We probably need to find a way to "chain" those - for example, if we want to have async tests with option sets. Your call whether to do it in this PR or in a different one, though.